src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -373,6 +373,13 @@ void GraphOptimizer::FuseConvolutionMatMulDeconvAndBias(Graph& graph) {
         if (biasNode->getType() != Type::Input || !biasNode->isConstant() || biasNode->getChildEdges().size() != 1) {
             return false;
         }
+// ACL supports u32 bias only for int inputs
```

Comment:
Please align precisions with executor configuration: currently, here we have u32, in the configuration - i32